hazelcast-jet-sql/src/test/java/com/hazelcast/jet/sql/SqlAggregateTest.java

```diff
@@ -979,6 +980,17 @@ public void test_groupingSet() {
                 .hasMessageContaining("Function 'GROUPING SETS' does not exist");
     }
 
+    // Test detected behaviour described in https://github.com/hazelcast/hazelcast-jet/issues/2940 .
+    // Fixed in #2974.
+    @Ignore
```

Comment:
Is the point of this test to be run manually? Since it's ignored and doesn't do any assertions.

Reply:
Yes, because it reproduces log messages in terminal.

Reply:
I'd remove this test. It will never fail, nobody will ever run it. Also it doesn't belong to this class. For example, there's `JetSqlResultImplTest.when_closed_then_iteratorFails` which is very similar to this one.